lib/beaker/command.rb

```diff
@@ -148,9 +148,17 @@ def environment_string_for host, env
         env_array << "#{key.to_s.upcase}=\"#{val}\""
       end
 
-      environment_string = env_array.join(' ')
-
-      "env #{environment_string}"
+      if host['communicator'] =~ /bitvise/
```

Comment:
An 'is_cygwin' sounds fine to me - but you wouldn't be able to run that until you have an ssh connection established, which then runs us back into the circle of what sort of platform are we running on?

I think that we could add a var to the hosts file to indicate is_cygwin per-host and then check on that.  When this is merged I could do an update to all existing windows host files to make use of that var - or perhaps have it start as default to true and then later on we switch to default is false?